src/components/MediaPlayer/VideoJS/VideoJSPlayer.js

```diff
@@ -415,6 +415,13 @@ function VideoJSPlayer({
             item: nextFirstItem,
             type: 'switchItem',
           });
+        } else if (nextFirstItem.isEmpty) {
```

Comment:
`isEmpty` indicates whether the section/Canvas the `currentNavItem` belongs to is empty or not. And `null` says that, there are no active structure items.

In this use-case we identify the empty section/Canvas and reset `isEnded` flag since Ramp doesn't instantiate a new VideoJS instance  for the empty Canvas where we usually reset the `isEnded` flag in the state.